Makefile

```diff
@@ -0,0 +1,26 @@
+ifeq ($(shell uname),Darwin)
+    LDFLAGS := -Wl,-dead_strip
```

Comment:
I don't think you get `epoll(7)` on Darwin so you won't be able to compile there anyway.